src/Attribute.zig

```diff
@@ -0,0 +1,279 @@
+const std = @import("std");
+const mem = std.mem;
+const Tree = @import("Tree.zig");
+const NodeIndex = Tree.NodeIndex;
+
+const Attribute = @This();
+
+/// Unknown attributes have null tag
+tag: ?Tag,
```

Comment:
I don't think anything should be emitted for unknown attributes, it's not like we can do anything meaningful with them.

Reply:
Updated it to not emit anything for those. I also switched it to storing the `TokenIndex` instead of the parsed tag because that made it easier to consume unknown/ignored attributes.